Sources/NIO/System.swift

```diff
@@ -204,6 +205,10 @@ internal enum Posix {
     static let IPTOS_ECN_ECT0: CInt = CNIODarwin_IPTOS_ECN_ECT0
     static let IPTOS_ECN_ECT1: CInt = CNIODarwin_IPTOS_ECN_ECT1
     static let IPTOS_ECN_CE: CInt = CNIODarwin_IPTOS_ECN_CE
+    static let IP_RECVPKTINFO: CInt = CNIODarwin.IP_RECVPKTINFO
```

Comment:
Hi @Lukasa , @Austinpayne :

Sorry for adding comment in this merged PR.

Background:
I am using `Bazel` to build `NIOPosix` as static framework, but I got error - `"module 'CNIODarwin' has no member named 'IP_RECVPKTINFO'"`.
I didn't find where CNIODarwin exposes `IP_RECVPKTINFO`, and if I change to `Darwin.IP_RECVPKTINFO`, building is ok.
so why did you use `CNIODarwin.IP_RECVPKTINFO` at here?

Thanks so much!

Reply:
`CNIODarwin` should expose it by way of an include of a system header file. Can you clarify what system you're building on, with what SDK and so on?

Reply:
Thanks for your reply.

I am an iOS developer, so I build it on mac os with iOS sdk 15, and `Command Line Tools` is `Xcode 13.1 (13A1030d)`

BTW, if I use Xcode to create a new project and add `swift-nio` in `Swift Package` way, everything is ok. But my company project is using `Bazel` to build these frameworks

Reply:
This seems like it's a Bazel issue then. It may be worth following up with the maintainers of the Swift builders for Bazel to find out what they're doing differently than Xcode.

Reply:
got it.

thanks for your comment